File: C/PRAK304-2210817210015-MuhammadIrfan.c
#include<stdio.h>
    int main(){
        int bil;
        printf("Masukkan bilangan cacah: "); scanf("%d", &bil);
        if (bil == 0){
            printf("Nol");}
            else if(bil>=1 && bil<=9){
            printf("Satuan");}
            else if(bil>=10 && bil<=19){
            printf("Belasan");}
            else if(bil>=20 && bil <100){
            printf("Puluhan");}
            else{
            printf("Anda Menginput Melebihi Limit Bilangan");}
        return 0;
    }

File: Python/PRAK304-2210817210015-MuhammadIrfan.py
bil = int(input("Masukkan bilangan cacah: "))
if bil == 0:
    print("Nol")
elif bil>=1 and bil<=9:
    print("Satuan")
elif bil>=10 and bil<=19:
    print("Belasan")
elif bil>=20 and bil <100:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")